### PR TITLE
Update the logging namespace - qemu (k-m)

### PR DESCRIPTION
**Pull request by luckyh**
Issue: https://github.com/autotest/tp-qemu/issues/3048

Signed-off-by: Xu Han <xuhan@redhat.com>

### CODE REVIEW

File: qemu/tests/multi_vms_with_stress.py [hunk 1]
```diff
@@ -47,7 +46,7 @@ def is_stress_alive():
         cmd = 'pgrep -xl stress'
         if not utils_misc.wait_for(lambda: re.search(
                 r'\d+\s+stress', process.system_output(
-                        cmd, ignore_status=True).decode()), 10):
+                    cmd, ignore_status=True).decode()), 10):
```
Comment (YongxueHong) [inline qemu/tests/multi_vms_with_stress.py:49]:
```suggestion
                        cmd, ignore_status=True).decode()), 10):
```

File: qemu/tests/multi_disk_random_hotplug.py [hunk 2]
```diff
@@ -247,9 +246,9 @@ def run_stress_dd():
 
     Monitor.CONNECT_TIMEOUT = params.get_numeric('connect_timeout', 60)
     BlockDevicesPlug.ACQUIRE_LOCK_TIMEOUT = params.get_numeric(
-            'acquire_lock_timeout', 20)
+        'acquire_lock_timeout', 20)
```
Comment (YongxueHong) [inline qemu/tests/multi_disk_random_hotplug.py:249]:
```suggestion
            'acquire_lock_timeout', 20)
```

File: qemu/tests/multi_disk_random_hotplug.py [hunk 3]
```diff
@@ -247,9 +246,9 @@ def run_stress_dd():
...
     BlockDevicesPlug.VERIFY_UNPLUG_TIMEOUT = params.get_numeric(
-            'verify_unplug_timeout', 60)
+        'verify_unplug_timeout', 60)
```
Comment (YongxueHong) [inline qemu/tests/multi_disk_random_hotplug.py:251]:
```suggestion
            'verify_unplug_timeout', 60)
```

File: qemu/tests/migration_with_block.py [hunk 4]
```diff
@@ -260,15 +259,15 @@ def ping_pong_migration(repeat_times):
                                                                 args, kwargs)
                 migration_thread.start()
                 if not utils_misc.wait_for(lambda: (
-                        bool(vm.monitor.query("migrate")) and (
+                    bool(vm.monitor.query("migrate")) and (
```
[the remark this answers is not in the document]
Comment (luckyh) [reply qemu/tests/migration_with_block.py:262]: @YongxueHong per our discussion, style issues will not be fixed this time, so I will take your suggestion, thanks!

**Comment by luckyh** [comment]
@YongxueHong have updated per your suggestions, thanks!

**Review by YongxueHong — APPROVED** [review]

**Pull request merged**